src/components/Form/story.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import Form from '.';
+import Readme from './README.md';
+
+storiesOf('Uncategorized/Form', module)
+  .addParameters({
+    readme: {
+      sidebar: Readme,
+    },
+  })
+  .add(
+    'Overview',
+    () => (
+      <Form
+        initialValues={{ nameInput: 'james r. cat' }}
```

Comment:
Just a thought: What should happen if no initial values are provided? Might there be a good way to default to "empty" values?
I guess this could be tough if we don't know up front whether an input will be text, radio, etc.